src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -1,18 +1,20 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import getProp from 'lodash/get';
+import sortCollection from 'lodash/sortBy';
```

Comment:
Can we call this `sortBy` to make it clear that it's a generic utility method?

Reply:
Sure, the reason I named it that is that it causes conflicts when using the keys provided from virtualized-table and de-structuring, but I can rename it in state.